examples/platform/silabs/SiWx917/matter_config.cpp

```diff
@@ -79,6 +82,51 @@ void SilabsMatterConfig::ConnectivityEventCallback(const ChipDeviceEvent * event
     SILABS_LOG("Scheduling OTA Requestor initialization")
 }
 
+#if SILABS_TEST_EVENT_TRIGGER_ENABLED
+static uint8_t sTestEventTriggerEnableKey[TestEventTriggerDelegate::kEnableKeyLength] = { 0x00, 0x11, 0x22, 0x33, 0x44, 0x55,
+                                                                                          0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb,
+                                                                                          0xcc, 0xdd, 0xee, 0xff };
+
+static int hex_digit_to_int(char hex)
+{
+    if ('A' <= hex && hex <= 'F')
+    {
+        return 10 + hex - 'A';
+    }
+    if ('a' <= hex && hex <= 'f')
+    {
+        return 10 + hex - 'a';
+    }
+    if ('0' <= hex && hex <= '9')
+    {
+        return hex - '0';
+    }
+    return -1;
+}
+
+static size_t hex_string_to_binary(const char * hex_string, uint8_t * buf, size_t buf_size)
```

Comment:
This is reinventing code the SDK already has, for what it's worth @ericzijian1994 .  That's bad in an example that you expect applications to follow...